README.md

```diff
@@ -1,163 +1,205 @@
 
 ### Table of contents:
 
-* [Requirements](#requirements)
-* [Installation](#installation)
-    * [Easy Install with Conda](#easy-install-with-conda)
-    * [Docker](#using-a-docker-image)
-    * [Conda Environment](#installing-from-source-in-a-conda-environment)
-    * [Windows](#installing-in-windows)
-* [FAQ and Troubleshooting](#faq-and-troubleshooting)
-* [Getting Started](#getting-started)
-* [Contributing to DeepChem](/CONTRIBUTING.md)
-    * [Code Style Guidelines](/CONTRIBUTING.md#code-style-guidelines)
-    * [Documentation Style Guidelines](/CONTRIBUTING.md#documentation-style-guidelines)
-    * [Gitter](#gitter)
-* [DeepChem Publications](#deepchem-publications)
-* [Examples](/examples)
-* [About Us](#about-us)
-* [Citing DeepChem](#citing-deepchem)
+- [Requirements](#requirements)
+- [Installation](#installation)
+  - [Install latest package with conda](#install-via-conda-recommendation)
+  - [Install latest package with pip (WIP)](#install-via-pip-wip)
+  - [Install from source](#install-from-source)
+    - [General installation](#general-installation)
+    - [Use powershell (Windows)](#use-powershell-windows)
+  - [Install using a Docker (WIP)](#install-using-a-docker-with-gpu-wip)
+- [FAQ and Troubleshooting](#faq-and-troubleshooting)
+- [Getting Started](#getting-started)
+- [Contributing to DeepChem](/CONTRIBUTING.md)
+  - [Code Style Guidelines](/CONTRIBUTING.md#code-style-guidelines)
+  - [Documentation Style Guidelines](/CONTRIBUTING.md#documentation-style-guidelines)
+  - [Gitter](#gitter)
+- [DeepChem Publications](#deepchem-publications)
+- [Examples](/examples)
+- [About Us](#about-us)
+- [Citing DeepChem](#citing-deepchem)
 
 ## Requirements
-* [pandas](http://pandas.pydata.org/)
-* [joblib](https://pypi.python.org/pypi/joblib)
-* [sklearn](https://github.com/scikit-learn/scikit-learn.git)
-* [numpy](https://store.continuum.io/cshop/anaconda/)
-* [tensorflow](https://www.tensorflow.org/)
+
+DeepChem requires these packages on any condition.
+
+- [joblib](https://pypi.python.org/pypi/joblib)
+- [pandas](http://pandas.pydata.org/)
+- [Pillow](https://pypi.org/project/Pillow/)
+- [scikit-learn](https://github.com/scikit-learn/scikit-learn.git)
+- [tensorflow](https://www.tensorflow.org/)
 
 ### Soft Requirements
-DeepChem has a number of "soft" requirements. These are packages which are needed for various submodules of DeepChem but not for the package as a whole.
 
-* [RDKit](http://www.rdkit.org/docs/Install.html)
-* [six](https://pypi.python.org/pypi/six)
-* [MDTraj](http://mdtraj.org/)
-* [PDBFixer](https://github.com/pandegroup/pdbfixer)
-* [Pillow](https://pillow.readthedocs.io/en/stable/)
+DeepChem has a number of "soft" requirements. These are packages which are needed for various submodules of DeepChem but not for the package as a whole.
 
-### Super easy install via pip3
+- [RDKit](http://www.rdkit.org/docs/Install.html)
+- [MDTraj](http://mdtraj.org/)
+- [PDBFixer](https://github.com/pandegroup/pdbfixer)

```

Comment:
You removed Pillow from the list of requirements?  We do still use it.

Reply:
I just moved Pillow to a hard requirement because it is directly imported. https://github.com/deepchem/deepchem/pull/1871#discussion_r432828282

Reply:
I think that was probably done by mistake.  It's only used in a few places that load images, so it doesn't need to be a hard requirement.  We can fix that in a separate PR though.